apps/agent/arapp.json

```diff
@@ -1,28 +1,25 @@
 {
+  "appName": "agent.aragonpm.eth",
```

Comment:
Similarly, I was thinking of removing aragonCLI as a dependency, but still using it to publish these frontends for now.